lib/util/device-types.coffee

```diff
@@ -0,0 +1,19 @@
+# copied from the @resin-io/device-types
+# because that has the run-time dependency on coffee-script
+
+find = (array, predicate) ->
+	for el in array
+		return el if predicate(el)
```

Comment:
I'm just a bit cautious of bringing extra deps when the necessity is such trivial